### PR TITLE
Karma: Add Test for Duplicating Page with animations

### LINKED ISSUE #5737: Karma: Add Tests for Duplicating Page
Add karma tests for duplicating page. I mainly want this to be tested on elements with any of the possible animations applied, but we may find a need for other tests in here around page duplication too.

We should be able to recycle a lot of this code from this karma test for copy and paste to test the outward facing api here:
https://github.com/google/web-stories-wp/pull/5736

### PR DESCRIPTION
**Pull request by samwhale**
## Context

FIXIT week

## Summary

Adds a karma test to make sure that duplicating a page with animations works as intended.

## Relevant Technical Choices

n/a

## To-do

n/a

## User-facing changes

none

## Testing Instructions

Tests should pass

### QA

- [x] This is a non-user-facing change and requires no QA

### UAT

n/a

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #5737

### CODE REVIEW

**Review by BrittanyIRL — APPROVED** [review]
Thanks for adding notes too to walk through what the test is doing!

File: assets/src/edit-story/karma/duplicate.cuj.karma.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      name: /Duplicate Page/,
+    });
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  it('should duplicate an animation', async () => {
+    // open effect chooser
```
Comment (embarks) [inline assets/src/edit-story/karma/duplicate.cuj.karma.js:53]: Thanks for the comments!

**Review by embarks — APPROVED** [review]

File: assets/src/edit-story/karma/copyAndPaste.cuj.karma.js [hunk 2]
```diff
@@ -164,7 +164,7 @@ function sequencedForEach(htmlCollection, op) {
           );
           expect(pasted.elementAnimations.length).toEqual(1);
 
-          // Coppied and Pasted anims should share all attributes
+          // Copied and Pasted animations should share all attributes
```
Comment (maxyinger) [inline assets/src/edit-story/karma/copyAndPaste.cuj.karma.js:167]: lol thanks for cleaning up my mess.

**Review by maxyinger — APPROVED** [review]
beautiful

**Pull request merged**